Samples/VShadowVolumeShadowCopy/cpp/shadow.cpp

```diff
@@ -674,6 +674,37 @@ int CommandLineParser::MainRoutine(vector<wstring> arguments)
 
 #endif
 
+
+        // Perform a support check
+        if (MatchArgument(arguments[argIndex], L"isup")) //wsSnapIdDest
+        {
+            ft.WriteLine(L"(Started support check)");
+
+            // Initialize the VSS client
+            m_vssClient.Initialize(VSS_CTX_ALL);
+
+            WCHAR wszVolumePathName[MAX_PATH];
+            BOOL supported = TRUE;
+            BOOL bWorked = ::GetVolumePathName(arguments[1].c_str(), wszVolumePathName, MAX_PATH);
+
+            HRESULT is_supported_result = m_vssClient.IsVolumeSupported(wszVolumePathName, &supported);
+
+            switch (is_supported_result)
+            {
+            case S_OK:
+                ft.WriteLine(L"\nSupported check is done, bool: %i, NESTED_VOLUME_LIMIT: %i", supported, false);
```

Comment:
Maybe create a separate value with?
L"\nSupported check is done, bool: %i, NESTED_VOLUME_LIMIT: %i"